migrations/create-plan.psql

```diff
@@ -5,6 +5,6 @@ CREATE TABLE IF NOT EXISTS plan (
   price                 INT8 NOT NULL,
   site_quota            INT8 NOT NULL,
   comment_quota         INT8 NOT NULL,
-  branded               BOOLEAN NOT NULL,
+  commercial            BOOLEAN NOT NULL,
```

Comment:
Migrations don't run unless you run them with `bin/migrate-deploy`, so this would never get automatically re-run. I've changed it because it's used by `bin/setup-db` to bootstrap a new development instance.